app/src/main/res/layout/consitraint_layout_tablet.xml

```diff
@@ -14,15 +14,15 @@
         android:layout_height="wrap_content"
         android:text="&lt;Back"
         android:textColor="@android:color/black"
-        android:textSize="20dp"
+        android:textSize="40dp"
```

Comment:
I recommend to use sp instead of dp to set textSize
Please check difference of sp and dp